docs/scripts_md/MakeArchiveLocationRelative.md

```diff
@@ -0,0 +1,53 @@
+# NAME
+
+MakeArchiveLocationRelative.pl -- Removes the root directory from the
+`ArchiveLocation` field in the `tarchive` table to make the path to the
+tarchive relative.
+
+# SYNOPSIS
+
+perl MakeArchiveLocationRelative.pl `[options]`
+
+Available option is:
+
+\-profile: name of the config file in ../dicom-archive/.loris\_mri
+
+# DESCRIPTION
+
+This script will remove the root directory from the ArchiveLocation field
+in the tarchive table to make path to the tarchive relative. This should
+be used once, when updating the LORIS-MRI code.
```

Comment:
Isn't this ambiguous? It was needed for the 15.10 release or even before :)

Reply:
Changed it to: 

"This should
be used once to remove the root directory if the C<tarchive> table still has 
some C<ArchiveLocation> paths stored from the root directory."